manual/manual.tex

```diff
@@ -173,14 +173,15 @@ \subsection{Off-Target Enumeration}
     \hline
     Option            & Description                                                                                                                                                                                                                                                                                         \\ \hline
     \texttt{--start}           & \begin{tabular}[c]{@{}l@{}}Specify whether the PAM should be matched at the start or end of the\\ protospacer sequence. The default is FALSE, in which the PAM is at\\ the 3' end of the protospacer.\end{tabular}                                                                                  \\ \hline
-    \texttt{--max-off-targets} & \begin{tabular}[c]{@{}l@{}}Maximum number of off-targets to store at each mismatch distance. If \\ this takes a negative value, all off-targets are enumerated at that distance.\\ The default value is -1, in which all off-targets are enumerated.\end{tabular}                                   \\ \hline
+    \texttt{--max-off-targets} & \begin{tabular}[c]{@{}l@{}}Maximum number of off-targets to store at each mismatch distance. If \\ this takes a negative value, all off-targets are enumerated at that distance.\\ The default value is -1, in which all off-targets are enumerated.\\ Note that specificity value of a match is calculated w.r.t the off-targets, so\\ specifying this option will modify the calculated specificity value as well.\end{tabular}                                   \\ \hline
```

Comment:
I thought this was an important point to clarify. Not sure if it should be more prominent.